stock_picking_no_over_scan/models/stock.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import models, _
+from odoo.exceptions import ValidationError
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    def _prepare_extra_move_vals(self, qty):
```

Comment:
@bodedra : Hi Bhavesh, are you sure with direct raise statement in this prepare_extra_move_vals method?

Reply:
@smangukiya Yes. I have do analysis and this method called when we scan more quantity than needed. Original move will be split with two section, one with done quantity and second with remaining quantity. We don't want to scan more quantity so direct raise. Please correct me if we need to consider any other side or workflow.